smoke-tests/framework/src/main/resources/default_applicationinsights.json

```diff
@@ -0,0 +1,9 @@
+{
+  "role": {
+    "name": "testrolename",
+    "instance": "testroleinstance"
+  },
+  "sampling": {
+    "percentage": 100
```

Comment:
the default is now limitPerSecond `5`, which can cause smoke tests to sporadically fail if they run more requests than that per second, so this is needed across all smoke tests, except for smoke tests which specifically test sampling

Reply:
make sense to me.. i was curious to know why :)